nservicebus/operations/performance-counters_counters_core_[6].partial.md

```diff
@@ -0,0 +1,7 @@
+include: performance-counters-criticaltime
+
+include: performance-counters-note
+
+include: performance-counters-slaviolation
```

Comment:
@WojcikMike I think removing the include makes most sense, it'll also simplify the structure a bit ;)